cluster/gce/gci/configure-helper.sh

```diff
@@ -279,6 +279,38 @@ contexts:
   name: webhook
 EOF
   fi
+
+if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
+    # This is the config file for the image review webhook.
+    cat <<EOF >/etc/gcp_image_review.config
```

Comment:
is there any particular reason these config files have to be under /etc rather than under ${KUBE_HOME}?